### ISSUE TITLE
Did u2net face generation was trained with the same loss function?

### ISSUE THREAD

**Issue by shgidi** [NathanUA/U-2-Net#97]

**Comment by NathanUA** [comment]
Yes， it is also trained with BCE.

**Comment by shgidi** [comment]
Thanks @NathanUA . Ine more question: did you use the same small dataset of 70 images?

**Comment by NathanUA** [comment]
We didn’t include the small set of 70 images in training. We only trained with their around 400 pairs of images in training without pretrained weights.
